.github/workflows/ci.yml

```diff
@@ -54,6 +54,7 @@ jobs:
         uses: infinyon/fluvio@master
         with:
           cluster-type: local
+          version: 'latest'
```

Comment:
I have a feeling that doing this will come back to bite us but meh, fine for now.